composer.json

```diff
@@ -41,9 +43,14 @@
     "autoload-dev": {
         "psr-4": { "Doctrine\\Bundle\\MigrationsBundle\\Tests\\": "Tests" }
     },
+    "config": {
+        "platform": {
+            "php": "7.2.22"
```

Comment:
Good call adding this while we commit the lock file. However, in the interest of testing more sensibly I'd like to refactor this in a separate pull request and test without a committed lock file similarly to how it's done in DoctrineBundle.

Reply:
this was just a mistake, I will rever it